lib/wallets/wallet/impl/monero_wallet.dart

```diff
@@ -218,8 +277,8 @@ class MoneroWallet extends CryptonoteWallet with CwBasedInterface {
           final addressString =
               (CwBasedInterface.cwWalletBase as MoneroWalletBase?)
                   ?.getTransactionAddress(
-            addressInfo!['accountIndex'] as int,
-            addressInfo['addressIndex'] as int,
+            addressInfo?['accountIndex'] as int? ?? 0,
+            addressInfo?['addressIndex'] as int? ?? 0,
```

Comment:
Is it ok to default to 0 here? Or should there be some error?

Reply:
should be fine, there is no account switching feature in stack yet.